Cargo.toml

```diff
@@ -51,6 +51,8 @@ dirs = "4.0.0"
 
 openssh-mux-client = { version = "0.15.0", optional = true }
 
+openssh-sftp-client = { version = "0.9.0", optional = true }
```

Comment:
I want to avoid bringing this in as a dependency — it _shouldn't_ be necessary, since nothing in SSH depends on SFTP (it's the other way around).

Reply:
It can be a `dev-dependency` so we can use it in tests/examples though!

Reply:
Done!